documentation/general/dotnet-run-file.md

```diff
@@ -172,56 +172,54 @@ App/Program2/bin/
 App/Program2/obj/
 ```
 
-## Package references
+## Directives for project metadata
 
-It is possible to specify NuGet package references via the `#package` directive.
+It is possible to specify some project metadata via [ignored C# directives][ignored-directives].
+Directives `sdk`, `package`, and `property` are translated into `<Project Sdk="...">`, `<PackageReference>`, and `<Property>` project elements, respectively.
+Other directives result in a warning, reserving them for future use.
 
 ```cs
-#package Newtonsoft.Json 13.0.1
+#:sdk      Microsoft.NET.Sdk.Web
+#:property TargetFramework=net11.0
+#:property LangVersion=preview
+#:package  System.CommandLine=2.0.0-*
 ```
 
-The C# language needs to be updated to ignore these directives (instead of failing the compilation).
-See [the corresponding language proposal][pound].
+The value must be separated from the name of the directive by white space and any leading and trailing white space is not considered part of the value.
```

Comment:
I'm definitely in favor of the package directive. I'm ok with the others, but I want to make sure that we have a clear end point. I don't mind adding things like this as they become necessary, but at some point, we should say "you seem to want a csproj without the csproj. Can you just use a csproj?"